Android--EN/Room.java

```diff
@@ -1149,13 +1149,13 @@ public void disableDeviceInputs(final boolean disableOperations) {
     /**
      * Sets the delay time for sending the whiteboard contents of the local user to the remote users.
      *
-     * After calling this method, the SDK delays sending the whiteboard contents of the local user with the remote users per the set time.
+     * After calling this method, the SDK delays sending the whiteboard contents of the local user to the remote users per the set time.
      *
      * This method helps synchronize the whiteboard contents with audio and video in CND live streaming.
      *
      * @note This method does not delays the content the local users see, that is, when the local user writes or draws on the whiteboard, they see the content on their own whiteboard immediately.
      *
-     * @param delaySec The delay time in seconds.
+     * @param delaySec The delay time in seconds. // TODO William Not milliseconds? It looks like it's being multiplied by 1000? Please doublecheck.
```

Comment:
The delay time that you pass in is the time in seconds. The SDK will multiply the set value by 1000 to turn it to time in milliseconds.

Reply:
Okay!